source/api/registry/profiles/index.md

```diff
@@ -50,10 +50,11 @@ The process for having a new entry added to this registry is [described here][re
 
 ## 3. Registry
 
-This table summarizes the known behaviors available, for use with the [Presentation API][prezi-api].
+This table summarizes the known profiles available, for use with the [Presentation API][prezi-api].
 
-| Behavior                       | 
+| Profile  | Description        | 
 | ------------------------------ |
+| http://www.loc.gov/mods/v3  | The URI for identifying [MODS (Metadata Object Description Schema)](http://www.loc.gov/standards/mods/index.html) metadata records.  |
```

Comment:
I think the documentation link should be https: https://www.loc.gov/standards/mods/mods-schemas.html

Reply:
I've made that change.